Front-end/src/Home.jsx

```diff
@@ -87,6 +87,8 @@ const Home = () => {
       let LastName;
       let Role;
       let Headline;
+      let Age;
+      let State;
```

Comment:
Why declaring them as let type? Could you do a const assignment directly?
i.e. const lastName = formData.get("LastName");

Also we should not capitalize the variable name.

Reply:
I think it was that I had a const function that I needed to change in the back-end and I subconsciously changed to let. I can update now.

Reply:
Actually I will leave this for tomorrow since I have a few questions I would like to bring up.

Reply:
Completed